src/main/java/com/artipie/api/RepositoryRest.java

```diff
@@ -73,17 +87,20 @@ public void init(final RouterBuilder rbr) {
      */
     private void getRepo(final RoutingContext context) {
         final RepositoryName rname = new RepositoryName.FromRequest(context, this.layout);
-        final ValidRepositoryName validator = new ValidRepositoryName(rname);
-        if (!validator.isValid()) {
+        final RepositoryNameValidator rnamecheck = new RepositoryNameValidator(rname);
+        if (!rnamecheck.isValid()) {
```

Comment:
There are a lot of code duplications.
We could create a default method in `Validator` (or create a private method here):
```
default boolean validate(final RoutingContext context, int error) {
        boolean valid = this.isValid();
        if (!valid) {
            context.response()
                .setStatusCode(error)
                .end(this.errorMessage());
        }
        return valid;
    }
```

Can validator return also error code?